### PR TITLE
Category list

### PR DESCRIPTION
**Pull request by obliviusm**
- add static list of Categories
- show list in side menu
- add list to state
- pass chosen category to products list

### CODE REVIEW

File: index.android.js [hunk 1]
```diff
@@ -10,26 +10,55 @@ import {
...
+    this.setState({
+      categoryIndex: index
+    });
+  }
+
   renderNavigationView() {
     return(
       <View style={styles.navigationView}>
-	<Text>Hello World</Text>
+	      <Text>Categories</Text>
```
Comment (marchi-martius) [inline index.android.js]: Please move whole stuff from here to new container.

File: index.android.js [hunk 2]
```diff
@@ -10,26 +10,55 @@ import {
...
 
   render() {
     return (
       <DrawerLayoutAndroid
-	drawerWidth={300}
-	renderNavigationView={this.renderNavigationView}>
-	<ProductsList />
+        	drawerWidth={300}
+        	renderNavigationView={this.renderNavigationView.bind(this)}>
+      	<ProductsList category={Categories[this.state.categoryIndex]} />
```
Comment (marchi-martius) [inline index.android.js]: `category={this.currentCategory}`

File: index.android.js [hunk 3]
```diff
@@ -10,26 +10,55 @@ import {
...
 
 import ProductsList from './app/containers/ProductsList';
 
+const Categories = [
+  {name: 'Antiques'},
+  {name: 'Books, Comics & Magazines'},
+  {name: 'Cars, Motorcycles & Vehicles'},
+  {name: 'Events Tickets'},
+  {name: 'Pet Supplies'}
+]
```
Comment (marchi-martius) [inline index.android.js]: Please move to external constants file

File: index.android.js [hunk 4]
```diff
@@ -10,26 +10,55 @@ import {
...
+      dataSource: ds.cloneWithRows(Categories),
+      categoryIndex: 0
+    };
+  }
+
+  onPressCategory(index) {
+    this.setState({
+      categoryIndex: index
+    });
+  }
```
Comment (marchi-martius) [inline index.android.js]: Whole stuff should be encapsulated in container.

**Review by marchi-martius — CHANGES_REQUESTED** [review]
Please take a look at my comments.

**Comment by obliviusm** [comment]
@marchi-martius moved some stuff. not sure if i can move `onPressCategory`

**Comment by marchi-martius** [comment]
@obliviusm please add tests

**Review by marchi-martius — APPROVED** [review]

**Pull request merged**